packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -11,8 +11,8 @@ dependencies: {{#enable_dev_menu}}
   alice: ^1.0.0-dev.12
   alice_dio: ^1.0.6{{/enable_dev_menu}}
   collection: ^1.18.0
-  copy_with_extension: ^5.0.4{{#enable_pin_code}}
-  crypto: ^3.0.6{{/enable_pin_code}}
+  copy_with_extension: ^5.0.4{{#has_authentication}}
+  crypto: ^3.0.6{{/has_authentication}}
```

Comment:
Do we still need the crypto package ?

Reply:
Not needed. Removed.

Reply:
We're still using crypto in several places:

![Screenshot 2024-12-23 at 16 45 06](https://github.com/user-attachments/assets/45bf7d79-6f7c-4b11-8b21-07558c8cd981)